rewards/boost/calc_boost.py

```diff
@@ -53,15 +55,19 @@ def get_badger_boost_data(stake_ratios: Dict) -> Tuple[Dict, Dict]:
     badger_boost_data = {}
     stake_data_ranges = {}
     for addr, stake_ratio in stake_ratios.items():
-        user_boost = 1
         user_stake_range = 0
+        if flags.flag_enabled("BOOST_STEP"):
+            user_boost = 1 if stake_ratio == 0 else min(math.floor(stake_ratio * 2000), 2000)
+        else:
+            user_boost = 1
         for stake_range, multiplier in STAKE_RATIO_RANGES:
-            if stake_ratio > stake_range:
-                user_boost = multiplier
+            if stake_ratio >= stake_range:
+                if not flags.flag_enabled("BOOST_STEP"):
```

Comment:
You can use constant instead of a string. 
`from rewards.feature_flags.feature_flags import BOOST_STEP`